Shakikka/refactorTractor
# Fix/Desktop Layout

## What does this PR do?

- This branch initially began to fix responsiveness, but I had to backtrack and tackle the layout/styling of desktop view
- The desktop view is now fully styled with grid
- Each info column now displays very rudimentary styling for dev readability
- Each info column now has its own corresponding SASS file
- CSS class blocks have been pulled from the original styles.css file and relocated to SASS files where applicable
- base.scss file now imports/compiles all CSS files and is itself imported into the scripts.js file
- I changed the colors too!

## How should it be tested in the UI?

- These changes will cause issues for displaying the API info --> html elements have been commented out and rearranged so the querySelectors might not have anything to pull from
- Desktop view should be easier to read and understand
- Responsiveness is still jacked.
